Src/BootCamp.Chapter/Command.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    public static class Command
```

Comment:
instead of having one static command class split each methods into a command of its' own. THen have a command factory.
Lastly, this class should simply be:
```
interface ICommand
{
  void Execute();
}
```
meaning which you just pass all the input through ctor and all commands can be executed in the same way.

Reply:
i think this is what you meant ?